spec/loadable_spec.rb

```diff
@@ -41,33 +41,11 @@
       end
     end
 
-    context 'With default validations disabled' do
-      before do
-        SitePrism.default_load_validations = false
-      end
-
-      after do
-        # Reset the Default Constructor so its behaviour can be modified
-        SitePrism::Page.remove_instance_variable(:@_load_validations)
-        SitePrism.default_load_validations = true
-      end
-
-      context 'A standard Page' do
-        it 'has no default load validations' do
-          klass = Class.new(SitePrism::Page)
-
-          expect(klass.load_validations.length).to eq(0)
-        end
-      end
-    end
+    context 'A standard Page' do
+      it 'has no default load validations' do
+        klass = Class.new(SitePrism::Page)
```

Comment:
This area I'm not entirely sure how to get it working the way I want. If you can point me in the right way I'll fix it up.

It "may" be that I can actually fix this now there is only one code path not 2 (0 / 1 default validations). For some of the Loadable tests here, they create phantom classes that just inherit the Loadable module (And are several years old), so I'm loathe to change them.

I'll have a look next week to see if I can standardise this.